server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -398,7 +420,6 @@ public void testLeaderDisconnectionDetectedQuickly() {
 
         cluster.stabilise(Math.max(
             // Each follower may have just sent a leader check, which receives no response
-            // TODO not necessary if notified of disconnection
```

Comment:
This TODO indicates that the timeout we calculate below can now be made shorter since we don't have to wait for an in-flight check to timeout. Same applies to the other TODOs like it.

Reply:
Oh wait, there are now tests of both cases, don't mind me.